hierarchicalforecast/evaluation.py

```diff
@@ -145,6 +145,50 @@ def msse(y, y_hat, y_train, mask=None):
     return loss
 
 # %% ../nbs/evaluation.ipynb 11
+def msse(y, y_hat, y_train, mask=None):
+    """Mean Squared Scaled Error
+
+    Computes Mean squared scaled error (MSSE), as proposed by Hyndman & Koehler (2006)
+    as an alternative to percentage errors, to avoid measure unstability.
+
+    $$ \\mathrm{MSSE}(\\mathbf{y}, \\mathbf{\\hat{y}}, \\mathbf{y}^{in-sample}) =
+    \\frac{\\frac{1}{h} \\sum^{n+h}_{t=n+1} (y_t - \\hat{y}_t)^2}{\\frac{1}{n-1} \\sum^{n}_{t=2} (y_t - y_{t-1})^2},$$
```

Comment:
Can we change this equation:
$$ \\mathrm{MSSE}(\\mathbf{y}, \\mathbf{\\hat{y}}, \\mathbf{y}^{in-sample}) =
  \\frac{\\frac{1}{h} \\sum^{n+h}_{t=n+1} (y_t - \\hat{y}_t)^2}{\\frac{1}{n-1} \\sum^{n}_{t=2} (y_t - y_{t-1})^2},$$

To match Nixtla's losses notation:
$$ \\mathrm{MSSE}(\\mathbf{y}, \\mathbf{\\hat{y}}, \\mathbf{y}^{in-sample}) =
  \\frac{\\frac{1}{h} \\sum^{t+h}_{\\tau=t+1} (y_{\\tau} - \\hat{y}_{\\tau})^2}{\\frac{1}{t-1} \\sum^{n}_{\\tau=2} (y_{\\tau} - y_{\\tau-1})^2},$$

Reply:
Updated with some minor changes.